Editor/IO/Files/CopyFilesToDirectory.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace UniTools.Build
+{
+    [CreateAssetMenu(
+        fileName = nameof(CopyFilesToDirectory),
+        menuName = MenuPaths.IO + nameof(CopyFilesToDirectory)
+    )]
+    public sealed class CopyFilesToDirectory : BuildStep
+    {
+        [SerializeField] private PathProperty[] m_filePaths = default;
+        [System.Obsolete, HideInInspector]
+        [SerializeField] private PathProperty m_filePath = default;
+        [SerializeField] private PathProperty m_destination = default;
+
+        public override async Task Execute()
```

Comment:
Check is the m_sourceDirectory path is a directory path. If not - throw an exception

Reply:
Good idea. I will make sure `m_destinationDirectory` is a directory path.